index.html

```diff
@@ -43,7 +40,7 @@ <h2 class="connect" id="connect">LET'S CONNECT</h2>
             </div>    
         </section>  
     </div>
-    <section id="none" class="works">
+    <section class="works">
```

Comment:
- [optional]great job making the changes. please try to centralize your text beside the image and make the heading bolder as well like in the figma design
yours
![Screenshot (1062)_LI](https://github.com/AbednegoKibitok/module1microverse/assets/65298048/f5ce6ba8-08f5-4696-9ab5-75d27b961d5e)
figma
![Screenshot (1061)](https://github.com/AbednegoKibitok/module1microverse/assets/65298048/014b3e29-b127-4845-9302-302e489564a3)